src/pages/index.js

```diff
@@ -104,25 +148,78 @@ function closeModal(modal) {
 
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = nameInput.value;
-  profileDescription.textContent = descriptionInput.value;
-  closeModal(editModal);
+  setButtonText(submitButton, true, "Saving...");
+
+  api
+    .editUserInfo({
+      name: nameInput.value,
+      about: descriptionInput.value,
+    })
+    .then((user) => {
+      profileName.textContent = user.name;
+      profileDescription.textContent = user.about;
+      editFormElement.reset();
+      resetValidation(
+        editFormElement,
+        [nameInput, descriptionInput],
+        validationConfig
+      );
+      closeModal(editModal);
+      disableButton(editFormElement.querySelector(".modal__submit-button"));
+    })
+    .catch(console.error);
+  setButtonText(submitButton, true, "Save");
 }
 
 function handleAddCardSubmit(evt) {
   evt.preventDefault();
-  const inputValues = { name: cardNameInput.value, link: cardLinkInput.value };
-  const cardElement = getCardElement(inputValues);
-  cardsList.prepend(cardElement);
-  cardForm.reset();
-  disableButton(cardSubmitButton, settings);
-  closeModal(cardModal);
+  const cardData = {
+    name: cardNameInput.value,
+    link: cardLinkInput.value,
+  };
```

Comment:
*  According to the task, you need to change submit buttons text (`Saving…`) while waiting for the server response. And the user should see it.


Please, use `setButtonText` when you submit the card form as well